lib/views/pages/post/post_page.dart

```diff
@@ -0,0 +1,81 @@
+import "package:flutter/material.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/models/ui/post_overview.dart";
+import "package:proxima/viewmodels/post_view_model.dart";
+import "package:proxima/views/navigation/leading_back_button/leading_back_button.dart";
+import "package:proxima/views/pages/post/post_page_widget/bottom_bar_add_comment.dart";
+import "package:proxima/views/pages/post/post_page_widget/comment_list.dart";
+import "package:proxima/views/pages/post/post_page_widget/complete_post_widget.dart";
+
+class PostPage extends HookConsumerWidget {
+  static const postDistanceKey = Key("postDistance");
+  static const completePostWidgetKey = Key("completePostWidget");
+  static const commentListWidgetKey = Key("commentListWidget");
+  static const bottomBarAddCommentKey = Key("bottomBarAddComment");
+
+  const PostPage({
+    super.key,
+    required this.postOverview,
+  });
+
+  final PostOverview postOverview;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    ThemeData themeData = Theme.of(context);
+
+    final comments = ref.watch(commentListProvider);
+
+    List<Widget> appBarContent = [
+      const Text("Post"),
```

Comment:
You should name the string constants that you use and put them at the top of the class, so that they can easily be changed.